setup.cfg

```diff
@@ -40,7 +40,6 @@ install_requires =
   pandas>=0.23.0
   astropy>=3.2
   parfive[ftp]>=1.1.0
-  importlib_resources;python_version<"3.7"
```

Comment:
Are we backporting this?

Reply:
To 2.0?

Reply:
Has this gone because we now depend on >=3.7?

Reply:
We used to import this in our init file to get the sunpy version, now we don't anymore.